modules/acquia_cms_video/config/dx8/packages.yml

```diff
@@ -1,3 +1,4 @@
 - config/dx8/cohesion_content_templates.media_video_embedded.yml_
 - config/dx8/cohesion_content_templates.media_video_full.yml_
 - config/dx8/cohesion_content_templates.media_video_video_component.yml_
+- config/dx8/cohesion_content_templates.media_image_embedded.yml_
```

Comment:
@panshulK - If we are including this shouldn’t we also include for existing site install? using may be update hook.
cc @vishalkhode1

Reply:
@chandan-singh7929 as discussed the update hook is not required because the file added in acquia_cms_video module is duplicate and has to be removed.